tests/migrations/test_commands.py

```diff
@@ -695,6 +695,40 @@ def test_sqlmigrate_for_non_transactional_databases(self):
             self.assertNotIn(start_transaction_sql.lower(), queries)
         self.assertNotIn(connection.ops.end_transaction_sql().lower(), queries)
 
+    @override_settings(MIGRATION_MODULES={"migrations": "migrations.test_migrations_squashed"})
+    def test_sqlmigrate_ambigious_squashed_migration_name(self):
+        out = io.StringIO()
+        msg = "More than one migration matches '0001' in app 'migrations'. Please be more specific."
+        with self.assertRaisesMessage(CommandError, msg):
+            call_command("sqlmigrate", "migrations", "0001", stdout=out)
+
+    @override_settings(MIGRATION_MODULES={"migrations": "migrations.test_migrations_squashed"})
+    def test_sqlmigrate_replaced_migration(self):
+        out = io.StringIO()
+        call_command("sqlmigrate", "migrations", "0001_initial", stdout=out)
+        output = out.getvalue().lower()
+        self.assertNotEqual(output.find('-- create model author'), -1)
```

Comment:
n.b. just noticed these tests could also use `assertIn` / `assertNotIn` rather than `find()`. But it seems the tests in this file mix the two, so no worries.